editor/store.js

```diff
@@ -21,9 +21,10 @@ const GUTENBERG_PREFERENCES_KEY = `GUTENBERG_PREFERENCES_${ window.userSettings.
 /**
  * Creates a new instance of a Redux store.
  *
+ * @param {?Object} initialState Optional initial state
  * @return {Redux.Store} Redux store
  */
-function createReduxStore() {
+function createReduxStore( initialState ) {
```

Comment:
Would it make sense to rename it to `preloadedState` to match with Redux API? See: https://github.com/reactjs/redux/blob/e58e207213161042cb5853ddb6acb0d65034c3b0/src/createStore.js#L39.